web/templates/component/label.html.eex

```diff
@@ -1,3 +1,9 @@
+<%= if assigns[:params] do %>
+<a href="/?<%= assigns[:params] %>" class="link">
+<% end %>
 <p class="normal br2 tc ph1 pv1 dib mv0 f6" style="background-color:<%= assigns[:label].label_hex %>; color:<%= assigns[:label].font_hex %>">
   <%= assigns[:label].label_name %>
 </p>
+<%= if assigns[:params] do %>
+</a>
+<% end %>
```

Comment:
This seems a little awkward, I can't think of a nicer way to do this though.

I've opened a stackoverflow question on this to see if there may be a better way: https://stackoverflow.com/questions/46277008/wrapping-html-elements-given-a-condition-in-phoenix-templating

Reply:
Yeah I didn't feel clean writing it...

Thanks Dan Smith (?)